OpenRA.Mods.Common/Activities/ExternalCaptureActor.cs

```diff
@@ -50,7 +48,9 @@ public override Activity Tick(Actor self)
 				return ActivityUtils.SequenceActivities(new MoveAdjacentTo(self, target), this);
 
 			if (!capturable.CaptureInProgress)
-				capturable.BeginCapture(self);
+			{
```

Comment:
Braces around single statements should be removed according to https://github.com/OpenRA/OpenRA/wiki/Coding-Standard.

Reply:
yep sorry about that, I thought that the `if` and `else` should be braced similarly but no.